### DISCUSSION TITLE
Just for curiosity Full Zoom Meeting

### DISCUSSION THREAD

**Discussion by alanfermtz** [sircharlo/meeting-media-manager#1887] [Q&A] [answered]
How can I use this program for full zoom meetings?

What I mean is, can I use only one pc to manage the media and be on the camera and give comments in the meeting at the time? How you did it?

Im curious because during the pandemic, we didnt use this program, we didnt know it exist. I hope i can explain myself

**Comment by mtdvlpr** [comment, marked as answer]
Hi @alanfermtz, what I would do and did is use M3 with the "convert to MP4" feature which can be enabled in the settings, fetch the media in advance which would automagically convert all images to videos. Then in Zoom, you could easily share all media by using share screen > advanced > video. You'll see a file selector where you have to select the correct video from the M3 output path where media is stored. This would show the video instead of the Zoom for you. You could play the video, it gets shared and when you close the video, Zoom pops back up. No need to have M3 or anything else open. Just Zoom.
